core/unix/os.c

```diff
@@ -8765,11 +8765,24 @@ get_dynamo_library_bounds(void)
     check_start = dynamo_dll_start;
     dynamorio_libname = IF_UNIT_TEST_ELSE(UNIT_TEST_EXE_NAME, DYNAMORIO_LIBRARY_NAME);
 #    endif /* STATIC_LIBRARY */
+
+#    if !defined(STATIC_LIBRARY) && defined(LINUX)
```

Comment:
It seems like there's no need for the ifdef: other configs can use this same new code, right?

Reply:
I think I need to keep
#ifndef  STATIC_LIBRARY
    dynamorio_libname = IF_UNIT_TEST_ELSE(UNIT_TEST_EXE_NAME, dynamorio_libname_buf);
#endif